terraform/_versions.tf

```diff
@@ -1,4 +1,6 @@
 terraform {
+  backend "azurerm" {
+  }
   required_version = ">= 1.0.4"
```

Comment:
should we upgrade to 1.3.7?

Reply:
yes, wanted to get this done first and stable, then go back and do a version bump. Happy to just do it at the same time if you think